plugins/stronghold/guest-js/index.ts

```diff
@@ -298,7 +298,14 @@ export class Store {
       snapshotPath: this.path,
       client: this.client,
       key: toBytesDto(key),
-    }).then((v) => Uint8Array.from(v));
+    }).then((v) => {
+         if (v !== null || undefined) {
+          Uint8Array.from(v)
+         }
+         else {
+          null
+        }
```

Comment:
Something like this i guess
```suggestion
         if (v !== null || undefined) {
          return Uint8Array.from(v)
         }
         else {
          return null
        }
```

Reply:
done